waspc/packages/deploy/src/providers/fly/createDb/createDb.ts

```diff
@@ -0,0 +1,37 @@
+import { $, question } from 'zx';
+import { exit } from 'process';
+import { getTomlFilePaths, serverTomlExistsInProject, getInferredBasenameFromServerToml } from '../helpers/tomlFileHelpers.js';
+import { createDeploymentInfo } from '../DeploymentInfo.js';
+import { CreateDbOptions } from './CreateDbOptions.js';
+import { getCommandHelp, waspSays } from '../helpers/helpers.js';
+import { flyDeployCommand, flySetupCommand } from '../index.js';
+
+export async function createDb(region: string, options: CreateDbOptions): Promise<void> {
+	waspSays('Creating your DB on Fly.io!');
+
+	const tomlFilePaths = getTomlFilePaths(options);
+
+	if (!serverTomlExistsInProject(tomlFilePaths)) {
+		waspSays(`${tomlFilePaths.serverTomlPath} missing. Skipping DB creation. Perhaps you need to run "${getCommandHelp(flySetupCommand)}" first?`);
+		exit(1);
+	}
+
+	const inferredBaseName = getInferredBasenameFromServerToml(tomlFilePaths);
+	const deploymentInfo = createDeploymentInfo(inferredBaseName, region, options, tomlFilePaths);
+
+	// Creates a DB, waits for it to come up, then links it to the app.
+	// The attachment process shares the DATABASE_URL secret.
+	const createArgs = [
+		'--name', deploymentInfo.dbName,
+		'--region', deploymentInfo.region,
```

Comment:
I was creating the db in `maa` region (Chennai, India) by accident and it told me 

```
You must have a paid plan to scale up in maa
```

Reference: https://community.fly.io/t/unable-to-create-a-postgres-databse/9291/5

Reply:
Yeah, I'm not sure there is much we can do with their free plan pricing policies upfront, unfortunately.